src/server/project.ts

```diff
@@ -142,6 +154,35 @@ namespace ts.server {
             return this.cachedUnresolvedImportsPerFile;
         }
 
+        public static resolveModule(moduleName: string, initialDir: string, host: ServerHost, log: (message: string) => void): {} {

```

Comment:
@RyanCavanaugh Shouldn't `require` at least be abstracted behind a call to `sys`?